src/search/data_loader.cpp

```diff
@@ -118,6 +118,51 @@ std::vector<ahItem*> CDataLoader::GetAHItemsToCategory(uint8 AHCategoryID, int8*
             ItemList.push_back(PAHItem);
         }
     }
+
+    const char* noQtyQuery = "SELECT item_basic.itemid, item_basic.stackSize "
```

Comment:
I guess don't do anything given that you're just following the format above but this is a pain to paste into MariaDB.